planning/behavior_path_planner/src/utils/path_safety_checker/objects_filtering.cpp

```diff
@@ -240,15 +239,28 @@ std::vector<PoseWithVelocityStamped> createPredictedPath(
                                 ? ego_predicted_path_params->time_horizon_for_front_object
                                 : ego_predicted_path_params->time_horizon_for_rear_object;
   const double time_resolution = ego_predicted_path_params->time_resolution;
+  const double delay_until_departure = ego_predicted_path_params->delay_until_departure;
 
   std::vector<PoseWithVelocityStamped> predicted_path;
   const auto vehicle_pose_frenet =
     convertToFrenetPoint(path_points, vehicle_pose.position, ego_seg_idx);
 
-  for (double t = 0.0; t < time_horizon + 1e-3; t += time_resolution) {
-    const double velocity =
-      std::clamp(current_velocity + acceleration * t, min_velocity, max_velocity);
-    const double length = current_velocity * t + 0.5 * acceleration * t * t;
+  for (double t = 0.0; t < delay_until_departure + time_horizon + 1e-3; t += time_resolution) {
```

Comment:
This `delay_until_departure` term is necessary?

Reply:
@satoshi-ota 
Thanks!
You are right, this `delay_until_departure` is not necessary.

Reply:
@satoshi-ota 
changed in efee117ed8a6f8110af4057019d742cc9e9d2890